packages/cozy-pouch-link/src/jsonapi.js

```diff
@@ -22,12 +24,36 @@ export const normalizeDoc = (doc, doctype) => {
   if (normalizedDoc.rev) {
     delete normalizedDoc.rev
   }
+
+  normalizeLinks(normalizedDoc, doctype, client)
+
   return normalizedDoc
 }
 
+const normalizeLinks = (docRef, doctype, client) => {
+  if (doctype !== 'io.cozy.apps') {
+    return
+  }
+
+  const webLink = generateWebLink({
+    cozyUrl: client.getStackClient().uri,
+    slug: docRef.slug,
+    subDomainType: client.capabilities.flat_subdomains ? 'flat' : 'nested',
+    pathname: '',
+    hash: '',
+    searchParams: []
+  })
+
+  docRef.links = {
+    self: `/apps/${docRef.slug}`,
+    related: webLink,
+    icon: `/apps/${docRef.slug}/icon/${docRef.version}`
+  }
+}
+
 const filterDeletedDocumentsFromRows = doc => !!doc
 
-export const fromPouchResult = (res, withRows, doctype) => {
+export const fromPouchResult = (res, withRows, doctype, client) => {
```

Comment:
nit: When dealing with many params, I prefer a `({param1, param2,...})` API

Reply:
Done in #1524